Python/Tests/Utilities.Python/PythonTestExplorer.cs

```diff
@@ -147,19 +149,52 @@ public PythonTestExplorer(VisualStudioApp app, AutomationElement element)
             Assert.IsNotNull(Tests, "Tests list is null");
         }
 
+        public void ClearSearchBar() {
+            if (_searchBarElement != null) {
+                InsertTextUsingUIAutomation(_searchBarElement, "");
```

Comment:
with this being an `AutomationWrapper`, you can just do `_searchBarElement.SetValue(string.Empty)`